src/__forks__/Promise.native.js

```diff
@@ -0,0 +1,35 @@
+/**
+ *
+ * Copyright 2013-2014 Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Ah, I see https://github.com/facebook/react-native/blob/master/Libraries/Promise.js, which probably should be updated with BSD header. Let's do that here.